docs/source/conf.py

```diff
@@ -240,8 +240,9 @@
 
 # Intersphinx configuration
 intersphinx_mapping = {
-    'traits': ('http://docs.enthought.com/traits', None),
     'pyface': ('http://docs.enthought.com/pyface', None),
+    "python": ("https://docs.python.org/3", None),
+    'traits': ('http://docs.enthought.com/traits', None),
```

Comment:
This change was motivated by a cross reference to unittest which is then removed, I will revert changes to this file.